scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -188,6 +189,11 @@ class FossId(
     private val waitForResult: Boolean
     private val secretKeys = listOf("serverUrl", "apiKey", "user")
     private val namingProvider: FossIdNamingProvider
+    // A list of all scans created in an ORT run, to be able to delete them in case of error.
+    // The reasoning is that either all these scans are successful, either none is created at all (clean slate).
+    // A use case is that a ORT run is created regularly e.g. nightly, and we want to have exactly the same amount
+    // of scan for each package.
+    private val createdScans = mutableSetOf<String>()
```

Comment:
Thanks for the comment with background information.
Some typos:
* (clean slate) -> (clean state)
* a ORT run -> an ORT run
* amout of scan -> amount of scans

Reply:
Points 2 and 3 are addressed.
Point 1: I meant indeed "clean slate" :)